dvm-helper/dvm-helper.go

```diff
@@ -662,24 +662,35 @@ func getInstalledVersions(pattern string) []dockerversion.Version {
 
 func getAvailableVersions(pattern string) []dockerversion.Version {
 	gh := buildGithubClient()
-	tags, response, err := gh.Repositories.ListTags("docker", "docker", nil)
-	if err != nil {
-		warnWhenRateLimitExceeded(err, response)
-		die("Unable to retrieve list of Docker tags from GitHub", err, retCodeRuntimeError)
-	}
-	if response.StatusCode != 200 {
-		die("Unable to retrieve list of Docker tags from GitHub (Status %s).", nil, retCodeRuntimeError, response.StatusCode)
+	options := &github.ListOptions{PerPage: 10}
```

Comment:
How about we bump the page size to 100? The default when omitted is 30, but since we are always getting every release, may as well use the max # per page to minimize requests.